src/bicep/modules/policyAssignment.bicep

```
@@ -0,0 +1,42 @@
+targetScope = 'resourceGroup'
+
+param builtInAssignment string = ''
+param logAnalyticsWorkspaceName string
+param workspaceResourceGroupName string
+param policyAssignmentPrefix string = 'MLZ -'
+
+// Creating a symbolic name for an existing resource
+resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2021-06-01' existing = {
+  name: logAnalyticsWorkspaceName
+  scope: resourceGroup(workspaceResourceGroupName)
+}
+
+var policyDefinitionID = {
+  NIST: {
+    id: '/providers/Microsoft.Authorization/policySetDefinitions/cf25b9c1-bd23-4eb6-bd2c-f4f3ac644a5f'
+    parameters: json(replace(loadTextContent('policies/NIST-policyAssignmentParameters.json'),'<LAWORKSPACE>', logAnalyticsWorkspace.id))
+  }  
+  IL5: {
+    id: '/providers/Microsoft.Authorization/policySetDefinitions/f9a961fa-3241-4b20-adc4-bbf8ad9d7197'
+    parameters: json(replace(loadTextContent('policies/IL5-policyAssignmentParameters.json'),'<LAWORKSPACE>', logAnalyticsWorkspace.id))
+  }
+  CMMC: {
+    id: '/providers/Microsoft.Authorization/policySetDefinitions/b5629c75-5c77-4422-87b9-2509e680f8de'
+    parameters: json(replace(loadTextContent('policies/CMMC-policyAssignmentParameters.json'),'<LAWORKSPACE>', logAnalyticsWorkspace.properties.customerId))
+  }  
+}
+
+var modifiedAssignment = (environment().name =~ 'AzureCloud' && builtInAssignment =~ 'IL5' ? 'NIST' :  builtInAssignment)
```

This is what we want, right? If we're in AzureCloud and you specified IL5, we kick you over to NIST, otherwise let it be?

```suggestion
var modifiedAssignment = ( environment().name =~ 'AzureCloud' && builtInAssignment =~ 'IL5' ) ? 'NIST' : builtInAssignment
```

Maybe `userSuppliedAssignment` and `calculatedAssignment` are clearer than `builtInAssignment` and `modifiedAssignment`?